week-2/debug/1.js

```diff
@@ -1,8 +1,9 @@
 // Predict and explain first...
 
 function sum(a, b) {
-  return;
-  a + b;
+  return a + b;
 }
 
 console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+
+// This function will not work because there is a semi colon after the return which should not be there. 
```

Comment:
And what happens when the semi-colon is there immediately after the `return` keyword?

Reply:
It will return the answer as undefined 🙂